ghcide/src/Development/IDE/GHC/Compat/Plugins.hs

```diff
@@ -67,6 +68,12 @@ initializePlugins env = do
     pure $ hscSetFlags newDf env
 #endif
 
+-- Plugins aren't stored in ModSummary anymore since GHC 9.0, but this
```

Comment:
Use Haddock comments, this way we can see the documentation just by hovering over the identifier:

```suggestion
-- | Plugins aren't stored in ModSummary anymore since GHC 9.0, but this
```

Reply:
I also just noticed that it was changed in 9.2, not 9.0, so I'll have to update the comment